File: scala/SimpleAk/src/main/scala-2.11/mol/Message.scala
package mol

/**
 * Created by cqshinn on 2/13/16.
 */
case class Message(var multableBuffer: StringBuffer = new StringBuffer())
case class ImmutableMessage(val immutableType: String)
case class ParseArticle(url: String)
case class ParseHtmlArticle(url: String, htmlString: String)
case class HttpResponse(body: String)
case class ArticleBody(url: String, body: String)

File: java/SimpleAk/src/main/java/mol/Message.java
package mol;

import java.io.Serializable;

/**
 * Created by cqshinn on 2/13/16.
 */
public class Message implements Serializable{
    public final StringBuffer mutableBuffer;

    public Message(StringBuffer mutableBuffer){
        this.mutableBuffer = mutableBuffer;
    }
}
